### ISSUE TITLE
As a dispatcher I want the time range filter to have an "included in time range" mode

### ISSUE THREAD

**Issue by Atala** [coopcycle/coopcycle-web#4351]
### What is the current state
- each task has a delivery window (doneAfter this hour, to be done before this hour)
- the date range filter works on "overlap" mode : if there is an overlap between the task time window and the filter time window then the task will be displayed

### What is the problem
- you may have tasks that are doable during the whole day let's say 9 to 5
- so this task will basically overlap will all the time range window you may put in the filter, so it will always show

### Example use cases : 
 - you want to filter out to see only the tasks that needs to be done in the next hour
 - you want to see only the tasks that needs to be done during lunch for focusing on foodtech

### What is the proposition :
 - the time range filter window will have two modes "overlaps" and "includes" and you can switch between the two
 -  i think you can check both at the same time (checkboxes) but maybe to simplify things you can only have one of them activated

**Comment by Paul-Eraman-CoopCycle** [comment]
I think that the best solution to their problem is the filter by store, at least to start with. Im afraid this will be overly complex.

**Comment by Paul-Eraman-CoopCycle** [comment]
i agree that this is a good solution BUT im getting worried about all these filters buried behand modals. i think if we are going to keep adding filters and options (which i want) we need to make these filters and their status (active or not) visible directly in the map without opening up any other modals

**Comment by Atala** [comment]
agree

datadog could be an inspiration here : 
idea 1 : search bar for "power users"
![Image](https://github.com/user-attachments/assets/12d08fd9-0f6a-46ca-aff0-123c7905c378)


idea 2 toggable left side bar with all the filters

![Image](https://github.com/user-attachments/assets/c438c874-5651-4443-9324-8ae1f9c1d006)

@Paul-Eraman-CoopCycle actually what about the toggalble left side bar? looks superior to the modal and a quickwin to me

**Comment by Paul-Eraman-CoopCycle** [comment]
For now i dont think we need that side menu (though i like the idea, but id rather do a real design process first)

Atm the idea is to show what is or is not activated. could we just have little boxes like this image show up on the top of the map to show filters once they are activated with an X to deactivate them?

![Image](https://github.com/user-attachments/assets/24421768-7c30-4ee8-8b1f-7733404e7427)

**Comment by Atala** [comment]
yes, that some work as well (and some design i guess depending on the filters)

**Comment by Paul-Eraman-CoopCycle** [comment]
estimate?

**Comment by Atala** [comment]
1-1,5 day
